### DISCUSSION TITLE
请教：关于EasyWeChat5.x的问题，按照文档操作，莫名报错Call to undefined

### DISCUSSION THREAD

**Discussion by mantouchong** [w7corp/easywechat#2560] [Q&A] [answered]
$user = $app->oauth-user();
文档里这么用没问题。

到了我这边，就有问题了。
Call to undefined method Overtrue\Socialite\Providers\WeChat::user()

查了一下，好像很多人跟我一样的问题，说是easywechat锁版本导致的文件丢失。

但具体也不知道什么意思，有大佬能指点一下么。

我这在门口就被卡住了。

**Comment by overtrue** [comment, marked as answer]
文档上没有你说的 `$user = $app->oauth-user();`

https://easywechat.com/5.x/official-account/oauth.html#sdk-%E4%B8%AD-oauth-%E6%A8%A1%E5%9D%97%E7%9A%84-api

**Comment by mantouchong** [comment]
我知道原因了。
因为咱们的文档链接有问题，我现在看的是5.x的文档，但是在里面点击侧边栏看内容，点着点着就自己链接到4.x的文档里。
也就是5.x的文档内容跟4.x的文档内容是掺杂在一起的。

**Comment by GabbyMrH** [comment]
我也经常发现在5.x的文档点击左侧导航栏切换会变成4.x，得注意才行，官方还没更新。
